### ISSUE TITLE
Final ranking

### ISSUE THREAD

**Issue by DaiJunyan** [PHME-Datachallenge/Data-Challenge-2022#12]
Dear Organizer,

Does the leaderboard count as a final ranking?
Will test data be used to calculate scores and announce them before the competition ends?

Best regards,
Junyan

**Comment by dgiordan88** [comment]
Dear Junyan,
The current leaderboard is computed using the Training set. The **final ranking** will be computed using a **separate Test set**. 

We are considering the possibility to publish the leader board computed by using the Test set.
Keeps checking the repository for updates on this manner.

Best

Danilo

**Comment by sXperfect** [comment]
Dear Organizer,

Assume that in the following days the leaderboard will be evaluated based on the test set.
Which result of each team will be taken as the final result? The best result for each task on test data or the result of the models on the last day of the competition?

Best,
Yeremia

**Comment by marty90** [comment]
The final ranking will be computed uniquely on the **test set**.

**Comment by DaiJunyan** [comment]
Dear Organizer,

The current team ranking on the leaderboard is based on the average value of the three f1 scores. Will the final ranking also be based on this average value? During training, Task 2 and 3 have fewer data than Task 1. Is it fair if we apply equal weights to the three f1 scores?

Best regards,
Junyan
